src/box/xrow_update.c

```diff
@@ -113,8 +113,7 @@ xrow_update_read_ops(struct xrow_update *update, const char *expr,
 	update->op_count = mp_decode_array(&expr);
 
 	if (update->op_count > BOX_UPDATE_OP_CNT_MAX) {
-		diag_set(ClientError, ER_ILLEGAL_PARAMS,
-			 "too many operations for update");
+		diag_set(IllegalParams, "too many operations for update");
```

Comment:
Let's start error messages with a capital letter now, that we don't have the "Illegal parameters, " prefix.

Reply:
Discussed f2f. This is large work here and what is worse there will be a large manual merge when https://github.com/tarantool/tarantool/pull/9996 will be rebased on top this patch. Decided to keep this way.